server/controllers/session.controller.js

```diff
@@ -31,11 +31,6 @@ export function getSession(req, res) {
 }
 
 export function destroySession(req, res, next) {
-  req.logout((err) => {
-    if (err) {
-      next(err);
-      return;
-    }
-    res.json({ success: true });
-  });
+  res.clearCookie('sessionId');
```

Comment:
Hey @lindapaiste, I use `clearCookie` here...
I think this is the one of the optimized solution